README.md

```diff
@@ -1,2 +1,219 @@
-# saritasa-github-actions
-GitHub actions
+# Workflows
+## android
+## asp.net
+## ios
+
+## wordpress
+### ci-wordpress reusable workflow
+
+### Deploying
+
+In order to deploy, you need to perform some steps:
+
+- Create a new workflow file for deploying your app, for example, [`deploy.yml`](https://github.com/saritasa-nest/ceai-wordpress/blob/develop/.github/workflows/ci-prod.yml)
+- Place the created playbook in your repository for example, [`ci\ansible\deploy.yaml`](https://github.com/saritasa-nest/ceai-wordpress/blob/develop/ci/ansible/ansistrano-rollback.yml)
```

Comment:
@udaltsovra it is okay to share project as example, but please add here info how we must put files into `shared` folders.

Reply:
(just add some comments that we must be accurate with these files and why)

Reply:
Added information about the folder structure